### ISSUE TITLE
训练集精度问题

### ISSUE THREAD

**Issue by Rashomon5487** [deepinsight/insightface#298]
lr-batch-epoch: 0.1 3999 0
testing verification..
(12000L, 512L)
infer time 82.392
[lfw][4000]XNorm: 16.636743
[lfw][4000]Accuracy-Flip: 0.85000+-0.01972
testing verification..
(14000L, 512L)
infer time 95.22
[cfp_fp][4000]XNorm: 15.990457
[cfp_fp][4000]Accuracy-Flip: 0.64200+-0.01316
testing verification..
(12000L, 512L)
infer time 80.785
[agedb_30][4000]XNorm: 15.559318
[agedb_30][4000]Accuracy-Flip: 0.63983+-0.01315
saving 2
[4000]Accuracy-Highest: 0.63983
INFO:root:Saved checkpoint to "../model-r100_CombineFace/model-0002.params"
INFO:root:Epoch[0] Batch [4096]	Speed: 37.06 samples/sec	acc=0.000000
INFO:root:Epoch[0] Batch [4224]	Speed: 103.41 samples/sec	acc=0.000000
INFO:root:Epoch[0] Batch [4352]	Speed: 103.40 samples/sec	acc=0.000000
INFO:root:Epoch[0] Batch [4480]	Speed: 103.60 samples/sec	acc=0.000000
INFO:root:Epoch[0] Batch [4608]	Speed: 103.63 samples/sec	acc=0.000000
INFO:root:Epoch[0] Batch [4736]	Speed: 103.42 samples/sec	acc=0.000000
作者你好，请问作者，训练集精度一直都是0.000，这样是正常的吗，验证集上精度是一直在上升的

**Comment by nttstar** [comment]
batchsize可能太小了

**Comment by Rashomon5487** [comment]
batchsize目前设置的是128

**Comment by Lxyzgf** [comment]
@Rashomon5487 
Maybe the learning_rate is too big. Try to decrease the lr.

**Comment by bing1zhi2** [comment]
I  have the same problem.  Maybe it doesn't matter........

**Comment by clhne** [comment]
@bing1zhi2 
Just waiting....
util epoch > 20，and show me the result.
